src/Pos/Delegation/Listeners.hs

```diff
@@ -64,7 +63,9 @@ pskLightRelay = Data $ DataParams $ \pSk -> do
                logDebug $
                    sformat ("Generating delivery proof and propagating it to neighbors: "%build) pSk
                let proof = proxySign SignProxySK sk pSk pSk
-               addToRelayQueue (DataOnlyPM (pSk, proof))
+               -- FIXME seems like a mistake. Shouldn't the relay subsystem
```

Comment:
There is a bug in this logic. But not in the line `void $ propagateData sendActions sendMsg (DataOnlyPM MsgMPC (pSk, proof))` -- we send message different from what we receive. But it's a mess regarding `pure False`/`pure True`. Created https://issues.serokell.io/issue/CSL-1370